### ISSUE TITLE
DIEN MODEL unable to save due to Valueerror

### ISSUE THREAD

**Issue by pavansvn** [shenweichen/DeepCTR#227]
**Describe the bug(问题描述)**
I was able to build the model DIEN but unable to save the model when I am using use_negsampling is true . I suspect that the add loss function has an issue as model which is built after train i m unable to save it and is throwing Value Error saying min 
if use_negsampling:
        model.add_loss(alpha * aux_loss_1)

**To Reproduce(复现步骤)**

**Operating environment(运行环境):**
 - python version [e.g. 3.4, 3.6]
 - tensorflow version [e.g. 1.4.0, 1.12.0]
 - deepctr version [e.g. 0.7.1,]

**Additional context**
